### ISSUE TITLE
Cannot resolve webpack's "alias" and "extensions"

### ISSUE THREAD

**Issue by maple3142** [CompuIves/codesandbox-client#427]
Open [https://codesandbox.io/s/github/maple3142/WynnStats/](https://codesandbox.io/s/github/maple3142/WynnStats/) and get a "ModuleNotFoundError"
But inside `webpack.config.dev.js` has
```javascript
resolve: {
  alias: {
    vue: 'vue/dist/vue.runtime.esm.js',
    '@': path.resolve('./src')
  },
  extensions: ['*', '.js', '.vue', '.css']
}
```

**Comment by CompuIves** [comment]
Hey @maple3142! We don't parse Webpack configs yet, we're still looking for ways to approach this. However, since `@` is a default configuration, I'll add it to the preset. Should be live today!

**Comment by lbogdan** [comment]
`@` alias added in #225 (and fixed in #769 ).
